--- a/CMakeLists.txt	2025-04-02 15:59:50.464387580 +0200
+++ b/CMakeLists.txt	2025-04-02 16:00:06.691064986 +0200
@@ -74,7 +74,7 @@
 
 # a test target
 ADD_CUSTOM_TARGET(test
-                  ./run_tests.sh ${CMAKE_CURRENT_BINARY_DIR}/test/parsing/yajl_test
+                  ./run_tests.sh $<TARGET_FILE:yajl_test>
                   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/test/parsing)
 
 ADD_CUSTOM_TARGET(test-api ${CMAKE_CURRENT_SOURCE_DIR}/test/api/run_tests.sh
